_data/events.yml

```diff
@@ -215,6 +205,39 @@
   githubUrl: ""
   blogPost: ""
 
+- name: "BBQ en Boardgames Bonanza™"
+  date: 2024-08-30
+  hide: true
+  track: "FunEvent"
+  summary: >
+    BBQ + Boardgame Bonanza with Thomas Janssens was a hit! 
+    Itenium colleagues came together for an evening of grilling 
+    and board gaming. Games were competitive, and the atmosphere 
+    was relaxed, creating a fun break from the usual work routine. 
+    Whether it was rolling dice or flipping cards, everyone had a 
+    great time.
+
+  level: Introduction
```

Comment:
level is not necessary for FunEevent

**--> CREATE A NEW PR TO ADD THIS TO THE CHECKLIST SINCE IT IS THE xth TIME THIS HAPPENED**